pkg/services/blockfetcher/blockfetcher_test.go

```diff
@@ -0,0 +1,75 @@
+package blockfetcher_test
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/nspcc-dev/neo-go/pkg/config"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/services/oracle/neofs"
+	"github.com/nspcc-dev/neofs-sdk-go/client"
+	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	"github.com/nspcc-dev/neofs-sdk-go/object"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+	"github.com/nspcc-dev/neofs-sdk-go/user"
+	"github.com/stretchr/testify/require"
+)
+
+func TestService(t *testing.T) {
+	cfg := config.NeoFSBlockFetcher{
+		ContainerID: "9iVfUg8aDHKjPC4LhQXEkVUM4HDkR7UCXYLs8NQwYfSG",
+		InternalService: config.InternalService{
+			Enabled:      true,
+			UnlockWallet: config.Wallet{Password: "one", Path: "./"},
+		},
+		Addresses: []string{"st1.t5.fs.neo.org:8080"},
+		Timeout:   15 * time.Second,
+	}
```

Comment:
We can't use some real-network data in our unit-tests because I'd expect tests to pass even with no Internet connection or with bad NeoFS node availability. We need to rework this test in order not to depend on Internet access/NeoFS node response.